client/src/components/Orcid/index.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+
+import { Button, Image, Popup } from 'semantic-ui-react';
+
+type OrcidProps = {
+    identifier?: string;
+    name?: string;
+    login: () => void;
+    logout: () => void;
+};
+
+const OrcidLogin: React.FC<OrcidProps> = (props) => {
+    return (
+        <Popup
+            trigger={
+                <Button id='connect-orcid-button' onClick={props.login}>
+                    <img
+                        id='orcid-id-icon'
+                        src='src/images/ORCIDiD_iconvector.svg'
+                        alt='ORCID login button'
+                        width={24}
+                        height={24}
+                    />
+                    Authenticate with ORCID
+                </Button>
+            }>
+            <div>
+                <p>
+                    By logging in with ORCID, you will be able to update and
+                    vote on annotations in SCope.
+                </p>
+                <p>
+                    ORCID provides a persistent identifier that distinguishes
+                    you from other researchers and a mechanism for linking your
+                    research outputs and activities to this identifier.
+                    <br />
+                    Learn more at{' '}
+                    <a
+                        href='https://orcid.org/'
+                        target='_blank'
+                        rel='noopener noreferrer'>
+                        orcid.org
+                    </a>
+                </p>
+            </div>
+        </Popup>
+    );
+};
+
+const OrcidLogout: React.FC<OrcidProps> = (props) => {
+    return (
+        <Popup
+            trigger={
+                <Button id='connect-orcid-button' onClick={props.logout}>
+                    <Image
+                        id='orcid-id-icon'
+                        src='src/images/ORCIDiD_iconvector.svg'
+                        alt='ORCID logout button'
+                    />
```

Comment:
```suggestion
                        alt='ORCID logout button'
                        width={24}
                        height={24}
                    />
```
Need this here too otherwise the icon is huge:
![image](https://user-images.githubusercontent.com/5079892/127654089-4e5826f7-3486-4bcd-a28c-ac7776e361fa.png)